Samples/ForwardRenderer/ForwardRendererControls.cpp

```diff
@@ -114,6 +114,8 @@ void ForwardRenderer::applyAaMode(SampleCallbacks* pSample)
     // Release the TAA FBOs
     mTAA.resetFbos();
 
+    mLightingPass.pProgram->addDefine("_OUTPUT_MOTION_VECTORS");
```

Comment:
Do we really always need it?

Reply:
I had this for motion blur. Removed in latest commit.